OpenNefia.Core/Locale/LocalizationManager.cs

```diff
@@ -216,5 +223,47 @@ public bool TryGetLocalizationData(EntityUid uid, [NotNullWhen(true)] out LuaTab
 
             return TryGetTable($"OpenNefia.Prototypes.Entity.{metadata.EntityPrototype.ID}", out table);
         }
+
+        private void MungeEcksdee()
+        {
+            _entityCache.Clear();
+            return;
```

Comment:
Is this intentional?

Reply:
nope, opted for getting the data when it's needed rather than everything, so the way robust does it.
simply forgot to remove the function, is changed now.